### ISSUE TITLE
Sometimes  the Android talk client spams connections to the server

### ISSUE THREAD

**Issue by gradinaruvasile** [nextcloud/talk-android#283]
Sometimes my VM in which nexycloud is running has high cpu usage. If i look at the process list i see multiple (10?) php-fpm7.0 processes ( i have nginx and php-fpm). These processes all take some cpu time. What i saw in the access logs is that one of the Android nextcloud talk clients spams connections multiple times a second. All of them look like this:

```
192.168.1.13 - username [16/Aug/2018:01:03:17 +0300] "GET /nextcloud/ocs/v2.php/apps/spreed/api/v1/chat/nux8emok?lookIntoFuture=1&lastKnownMessageId=253&limit=25 HTTP/2.0" 200 669 "-" "Mozilla/5.0 (Android) Nextcloud-Talk v3.0.0beta3
```

The VM's cpu is pegged at 100%.

**Comment by mario** [comment]
Well, sure? Since we can't have a constantly open connection (PHP) there's no other way than poke it all the time. Did you set keepalive?

Sorry :(

**Comment by gradinaruvasile** [comment]
How can a situation where we have **multiple times a second**  connections open constantly and hogging **all** resources of a server regarded as normal?

This doesn't happen all the time. Most times i can have a/v calls without this spamming. This is not the standard behavior. If i kill the app, restart it and enter a conversation most likely this doesn't happen and i have very low resource usage without the functionality being affected in any way. It just happens sometimes and it really seems to be a bug and not a feature.

**Comment by gradinaruvasile** [comment]
> Can you tell me what's the keepalive interval of your server?
Where can i check that?

**Comment by mario** [comment]
Apache or nginx config.

**Comment by gradinaruvasile** [comment]
I have a "keepalive_timeout 65;" in the nginx.conf. Is that it? Seems to be a default value.

**Comment by mario** [comment]
That should be good enough hm

**Comment by mario** [comment]
Should be fixed in 3.0.0beta6.
